File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        if(s == " "){
            return true;
        }

        // s = s.replaceAll(" ", "");
        // s = s.replaceAll(",", "");
        // s = s.replaceAll(":", "");
        s = s.replaceAll("[^a-zA-Z0-9]", "");
        s = s.toLowerCase();
        System.out.println(s);
        StringBuilder sb = new StringBuilder(s);
        if(s.equals(sb.reverse().toString())){
            return true;
        }
        else{
            return false; 
        }
    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()
        # convert the string to lowercase
        s = ''.join(x for x in s if x.isalnum())
        # join only alphabets 
        return s == s[::-1]
        # if string s == reverse s return True